addons/mass_mailing/models/mailing.py

```diff
@@ -72,7 +89,7 @@ def _get_default_mail_server_id(self):
              'In most inboxes, this is displayed next to the subject.\n'
              'Keep it empty if you prefer the first characters of your email content to appear instead.')
     email_from = fields.Char(string='Send From', required=True,
-        default=lambda self: self.env.user.email_formatted)
+        default=lambda self: self._get_default_email_from())
```

Comment:
Shouldn't we make it a computed stored editable field instead ? That way if people change the server, email_from is updated.

Reply:
Yes, good idea :)